docs/docs/configuration/hardware_acceleration.md

```diff
@@ -286,10 +286,10 @@ These instructions were originally based on the [Jellyfin documentation](https:/
 
 ## NVIDIA Jetson (Orin AGX, Orin NX, Orin Nano\*, Xavier AGX, Xavier NX, TX2, TX1, Nano)
 
-A separate set of docker images is available that is based on Jetpack/L4T. They comes with an `ffmpeg` build
+We provide docker images that are based on Jetpack/L4T. They come with an `ffmpeg` build
```

Comment:
These docker images are not provided by us directly, they are community supported. I think this first sentence should remain unchanged

Reply:
Thank you, I have reverted the sentence back to its original.

If you don't mind, could you clarify what you mean by "not provided by us"? The image is part of the [Frigate project](https://github.com/blakeblackshear/frigate/tree/dev/docker/tensorrt), it seems that it is fair to use "we" here to refer to the repository/community/project as a collective, but not a big deal to me, I am just not sure what the distinction is.

Reply:
there are frigate maintainers and officially supported images and detectors, for example coral, openvino, tensorrt, etc.

Then there are community supported images and detectors like rknn and jetson. We as frigate maintainers do not provide user support or ongoing maintenance of these images - that is done by the code owner of the community supported image.

This is an important distinction to set expectations for the user, the images are maintained entirely by the community code owner and if it breaks or users have issues it is not something that we as the frigate maintainers can support with.

Reply:
Thank you! I appreciate the explanation.